eth_account/typed_transactions/access_list_transaction.py

```diff
@@ -227,9 +228,13 @@ def payload(self) -> bytes:
         """
         Returns this transaction's payload as bytes.
 
-        Here, the TransactionPayload = rlp([chainId,
-        nonce, gasPrice, gasLimit, to, value, data, accessList,
-        signatureYParity, signatureR, signatureS])
+        Here, the transaction payload is:
+
+        .. code-block:: python
+
+            TransactionPayload = rlp([chainId,
```

Comment:
The ``rlp`` python module can't be used in this way afaik. This may best be left as part of the docstring like it was. It provides a description of the rlp encoding, as it is usually defined in EIPs, rather than direct python code.

Reply:
Ah, you're right. Changed!